var/spack/repos/builtin/packages/py-grpcio/package.py

```diff
@@ -39,8 +43,18 @@ class PyGrpcio(PythonPackage):
     version("1.16.0", sha256="d99db0b39b490d2469a8ef74197d5f211fa740fc9581dccecbb76c56d080fce1")
 
     depends_on("py-setuptools", type="build")
-    depends_on("py-six@1.5.2:", when="@:1.48", type=("build", "run"))
-    depends_on("py-cython@0.23:2", type="build")
+    with default_args(type=("build", "run")):
+        depends_on("py-coverage@4:")
+        depends_on("py-wheel@0.29:")
+        depends_on("py-cython@3:", when="@1.63.0:")
+        depends_on("py-cython@0.29.8:", when="@1.49.0:1.62")
```

Comment:
`requirements.txt` for `1.56.2` and `1.62.2` imply `py-cython` range should stop at 2.* (e.g., `@0.29.8:2`).

```
cython>=0.29.8,<3.0.0rc1
```